landscape.yml

```diff
@@ -2117,6 +2117,19 @@ landscape:
             repo_url: https://github.com/traefik/mesh
             logo: traefik-mesh.svg
             crunchbase: https://www.crunchbase.com/organization/containous
+          - item:
+            extra:
+              dev_stats_url: https://smp.devstats.cncf.io/
+              artwork_url: https://github.com/cncf/artwork/blob/master/examples/sandbox.md#service-mesh-performance-logos
+              mailing_list_url: https://groups.google.com/a/smp-spec.io/g/community
+              slack_url: http://slack.layer5.io/
```

Comment:
note we will have to move the smp project Slack to CNCF' sslack

Reply:
Sure, will move that and update the link